src/lib/features/events/event-search-controller.ts

```diff
@@ -47,9 +47,10 @@ export default class EventSearchController extends Controller {
         this.eventService = eventService;
         this.flagResolver = config.flagResolver;
         this.openApiService = openApiService;
-        this.msgFormatter = new FeatureEventFormatterMd(
-            config.server.unleashUrl,
-        );
+        this.msgFormatter = new FeatureEventFormatterMd({
+            unleashUrl: config.server.unleashUrl,
+            formatStyle: 'markdown',
```

Comment:
We request properly formatted markdown only when necessary, minimizing the surface area of potential issues.

We can be confident that everything else remains unchanged and continues to meet previous expectations, as no updates to tests or snapshots were required.